src/command_modules/azure-cli-interactive/HISTORY.rst

```diff
@@ -3,9 +3,11 @@
 Release History
 ===============
 
-0.3.2 (unreleased)
+0.3.3 (unreleased)
```

Comment:
I think Derek prefers that you just leave this as "unreleased" and then he bumps the version number on release.